packages/fluentui/docs/src/prototypes/Roster/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import participants from './participantData';
+import presenters from './presentersData';
+import themeOverrides from './styles';
+import { Provider, Tree, Text, Flex, Button, Input, Header, teamsDarkTheme } from '@fluentui/react-northstar';
+import { CloseIcon, LinkIcon, TriangleDownIcon, TriangleEndIcon } from '@fluentui/react-icons-northstar';
+import { PrototypeSection, ComponentPrototype } from '../Prototypes';
+
+const titleRenderer = (Component, { content, expanded, open, hasSubtree, ...restProps }) => (
+  <Component expanded={expanded} hasSubtree={hasSubtree} {...restProps}>
+    {expanded ? <TriangleDownIcon /> : hasSubtree ? <TriangleEndIcon /> : ''}
+    <Text size="small" content={content}></Text>
+  </Component>
+);
+
+const treeItems = [
+  {
+    id: 'participants',
+    title: 'Currently in this meeting',
+    items: participants,
```

Comment:
items need to be defined as shorthand collection, not as a collection of components. Please see the Tree compoennt examples, for example in https://fluentsite.z22.web.core.windows.net/components/tree/definition?showCode=true&showRtl=false&showTransparent=false&showVariables=false#types-tree